### PR TITLE
Task/update UI homepage

### PR DESCRIPTION
**Pull request by nguyenhoaikhang37**
fix home page ui with dark mode

### CODE REVIEW

File: app/(auth)/sign-in/page.tsx [hunk 1]
```diff
@@ -1,21 +1,26 @@
 "use client"
 
-import { useAuthContext } from "@/context/auth"
```
Comment (nguyenhoaikhang37) [inline app/(auth)/sign-in/page.tsx:3]: @tranduybau I think we've different about prettier here. Can you setup prettier for uniformity?
`
// from
import { useAuthContext }
// to
import {useAuthContext}
`
or vice versa

File: next.config.mjs [hunk 2]
```diff
@@ -4,6 +4,9 @@ const nextConfig = {
   experimental: {
     appDir: true,
   },
+  images: {
+    domains: process.env.NEXT_PUBLIC_IMAGES_DOMAINS.split(","),
```
Comment (nguyenhoaikhang37) [inline next.config.mjs:8]: @tranduybau Can you explain for me this line, tks?

File: components/icons.tsx [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import {LogIn, Moon, SunMedium, LucideProps} from "lucide-react"
+import {LogIn, Moon, SunMedium, LucideProps, ArrowLeft} from "lucide-react"
```
Comment (nguyenhoaikhang37) [inline components/icons.tsx:1]: @tranduybau I think we need to separate 2 kind of icons (from svg and lucide-react). I wrote component Icon in branch `task/review-code-page`
how to use that component
`
<Icon
          name="arrow-left" // auto suggestion all of lucide icon name here
          className="absolute inset-y-0 left-3 mt-2 h-6 w-6 text-gray-500"
 />
 `

**Review by nguyenhoaikhang37 — COMMENTED** [review]
✅

File: components/icons.tsx (hunk 3 above)
Comment (tranduybau) [reply components/icons.tsx:1]: We will add eslint

**Review by tranduybau — APPROVED** [review]

**Pull request merged**